modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java

```diff
@@ -299,6 +299,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 
 			});
 
+		registry.register(
+			"3.9.1", "4.0.0",
```

Comment:
This is a micro change, so it should be 3.9.2

Reply:
Major changes

- Modifying data without backward compatibility.